crates/bws/src/main.rs

```diff
@@ -108,16 +108,15 @@ enum SecretCommand {
     Create {
         key: String,
         value: String,
+        project_id: Uuid,
 
         #[arg(long, help = "An optional note to add to the secret")]
         note: Option<String>,
-
-        #[arg(long, help = "The ID of the project this secret will be added to")]
```

Comment:
Did we intentionally drop the help text?

Reply:
I intentionally removed this line so that `--project-id` would no longer be passed via a command line flag but used as a parameter, like we do in other commands (ex: `secret edit`), especially as it is required. Is there a way to retain the help text?

Help menu output before and after this change below.

Before:
```
Usage: bws secret create [OPTIONS] --project-id <PROJECT_ID> <KEY> <VALUE>

Arguments:
  <KEY>    
  <VALUE>  

Options:
      --project-id <PROJECT_ID>
          The ID of the project this secret will be added to
      --note <NOTE>
          An optional note to add to the secret
...
```

After:
```
Usage: bws secret create [OPTIONS] <KEY> <VALUE> <PROJECT_ID>

Arguments:
  <KEY>         
  <VALUE>       
  <PROJECT_ID>  

Options:
      --note <NOTE>
          An optional note to add to the secret
...
```

Reply:
I think just removing the `long,` would make it into a positional argument, but I haven't tested it

Reply:
Ah, that seems to work! Thanks, fixed in [032f4e0](https://github.com/bitwarden/sdk/pull/77/commits/032f4e074656b5cce92ef562ecb77516e7c41701). This is the output now:

```
Usage: bws secret create [OPTIONS] <KEY> <VALUE> <PROJECT_ID>

Arguments:
  <KEY>         
  <VALUE>       
  <PROJECT_ID>  The ID of the project this secret will be added to

Options:
      --note <NOTE>
          An optional note to add to the secret
...
```